Adds support for creating wallet in a non interactive way (#218)
helium/docs: docs/mine-hnt/validators/wallet.mdx

```./helium-wallet create basic --network testnet``` 

#### Create or Use Wallet without Password Input

> For security reasons, it is generally recommended that you type in your password every time. Storing the password in an environmental variable makes it very easy for the password to get compromised. Sometimes though, despite the security risks, you may want to interact with the CLI without need to input the password. In this case, using the environmental variable `HELIUM_WALLET_PASSWORD` enables this.

Reviewer: ```suggestion
For security reasons, it is generally recommended that you type in your password every time. Storing the password in an environmental variable makes it very easy for the password to get compromised. Sometimes though, despite the security risks, you may want to interact with the CLI without inputting the password. In this case, using the environmental variable `HELIUM_WALLET_PASSWORD` enables this.
```

```

> HELIUM_WALLET_PASSWORD="My Password" ./helium-wallet create basic --network testnet

Reviewer: ```suggestion
HELIUM_WALLET_PASSWORD=p4ssw0rd ./helium-wallet create basic --network testnet
```

```./helium-wallet create basic --network testnet``` 

> #### Create a Wallet in a non-interactive / DevOps way

Reviewer: ```suggestion
#### Create or Use Wallet without Password Input
```

> 

Reviewer: ```suggestion
For security reasons, it is generally recommended that you type in your password every time. Storing the password in an environmental variable makes it very easy for the password to get compromised. Sometimes though, despite the security risks, you may want to interact with the CLI without need to input the password. In this case, using the environmental variable `HELIUM_WALLET_PASSWORD` enables this.
```

Author: The clarification is valid and explains why the variable exists, however I'd like to insist in specifying an example (like [this one](https://github.com/helium/docs/pull/218/files/4791a853daf3bccc8adc467a4ce93079a75e8a8f#diff-29853b474d408aac49fc4717bb1e6b78e0c1efc2d5b5b7c54c106a1926c962b1) so nobody is encouraged to use `export`.

Reviewer: This is the proposal. As I mentioned earlier, please accept the suggestions if you agree. I don't want to push to your branch if I can avoid it.

Author: @lthiery Agree to the modifications!

```

> export HELIUM_WALLET_PASSWORD="My Password"

Reviewer: ```suggestion

```

Author: That works too, although in this one-line version exporting the variable doesn't make a lot of sense (besides wanted a little more of visual clarity), but works for me!

Reviewer: I am suggesting removal of this line.

> ./helium-wallet create basic --network testnet

Reviewer: ```suggestion
HELIUM_WALLET_PASSWORD="My Password" ./helium-wallet create basic --network testnet
```

Reviewer: I would recommend this one  liner instead

Author: Precisely like this. The importance of not exporting HELIUM_WALLET_PASSWORD lies in the fact once the process has finished, the password will remain in memory. However, in the change proposed here (without the `export` command, the variable will be only availble to the helium-wallet process and nothing else.

Reviewer: I think we're in agreement. Removing the `export` line and injecting it during the executable invocation is what I've proposed  here. I'm waiting for you to accept the suggestions since it's your PR.
